src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/resources/PlatformResource.java

```diff
@@ -0,0 +1,44 @@
+package edu.kit.ipd.crowdcontrol.objectservice.rest.resources;
```

Comment:
It's currently consistent with all other classes in `rest.resources`. If we change it, it should be changed in a separate PR for all classes.